internal/controller/scheduler/scheduler.go

```diff
@@ -197,6 +197,9 @@ func (w *jobWrapper) Build() (*batchv1.Job, error) {
 		}, {
 			Name:  "BUILDKITE_BIN_PATH",
 			Value: "/workspace",
+		}, {	
+			Name:  "BUILDKITE_HOOKS_PATH",
+			Value: "/buildkite/hooks",
```

Comment:
I think the reason we didn't set this is that this directory won't necessarily exist. So setting it to some value won't work unless if users have some way to getting hooks into that directory.

If you're using a volume mount, then, you will need to do something at the step level to mount that volume into your container. (e.g. the example YAML in the file above). If you're going to the trouble of doing that, then you may as well specify `BUILDKITE_HOOKS` in that YAML. A user could mount their hooks volume anywhere, so it's not appropriate for the controller to inject a value for BUILDKITE_HOOKS that does not match the mount path.

So the example YAML should look something like:
```yaml
steps:
  - label: build image
    agents:
      queue: kubernetes
    plugins:
      - kubernetes:
          podSpec:
            containers:
              - image: gradle:latest
                command: [ gradle ]
                args:
                  - jib
                  - --image=ttl.sh/example:1h
                env:
                  - name: BUILDKITE_HOOKS
                    value: /buildkite/hooks  # <---- This value
            volumes:
              - name: agent-hooks
                configMap:
                  name: my-agent-hooks
                  defaultMode: 0755
          extraVolumeMounts:
            - name: agent-hooks
              mountPath: /buildkite/hooks   # <---- needs to be the same as this
```


If you're using a custom container image that contains these hooks, I think you should be able to put
```
ENV BUILDKITE_HOOKS /buildkite/hooks
```
in the Dockerfile. That might be more ergonomic.

Reply:
Sorry I missed this comment previously! I agree that someone could put their hooks anywhere but it seems like having a convention would be nice, particularly with the README example for new users.  But these options definitely work so I'm fine to drop it if you disagree.